LectureNotes/Lectures/Fall2018/Lecture6/ASPlecturenotes.tex

```diff
@@ -0,0 +1,142 @@
+\date{\today}
+\begin{document}
+\maketitle
+
+\section{The Cauchy Distribution}
+
+$C \sim Cauchy(1)$ 
+if $\mathbb{P}\{c \in dx\}= \dfrac{dx}{\pi(1+x)^2}$ and $t \times C \sim Cauchy(t)$
+
+\noindent\rule{4cm}{0.4pt} 
+
+\noindent$N \sim PPP(\dfrac{dtdx}{|x|^2})$ on $[0,\infty) \times \{ \mathbb{R}\setminus \{0\}\}$
+
+\noindent$Let C_t = ($sum of x-values of points of $[0,t] \times \mathbb{R})= \sum x \mathbbm{1}_{[0,t)}(t) dN(t,x)$ 
+
+\subsection*{Claim}
+$\dfrac{1}{t} C_t \stackrel{d}{=} C_1$
+
+\begin{proof}
+If $N = \sum \delta_{(t_i,x_i)} \Rightarrow \dfrac{1}{t} C_t = \sum_{i:t_i\leq1} \dfrac{x_i}{t}$ but \~N $= \sum_i \delta_{\left(\frac{t_i}{t_1}, \frac{x_i}{t}\right)} \sim PPP(\dfrac{dtdx}{x^2})$ i.e. $N \stackrel{d}{=} $\~N
+(since $\dfrac{dsdx}{x^2} = \dfrac{tdu tdv}{t^2dv}=\dfrac{dudv}{v^2}$, where $u=\frac{s}{t}$ and $v=\frac{x}{t}$), so $\dfrac{1}{t}C_t = \int x \mathbbm{1}_{[0,1]}(t) d\text{\~{N}} (t,x) = \text{\~C}(1)$ 
+\end{proof}
+
+
+\subsection*{Consequence}
+Let $x_1,x_2,...,x_n$ iid $Cauchy(1)$ then $\bar{x} = \dfrac{1}{n} (x_1+...+x_n) \sim Cauchy(1)$. This implies that that taking more samples does not tell you more about the mean! But ...\\
+$$\begin{aligned}
+var[\bar{x}] = \frac{1}{n}var[x_1] = \int_0^1 \int_{-\infty}^{\infty} x^2 \frac{dxdt}{x^2} = \infty \\ \text{ even } \mathbb{E}[x_1]=\int_0^1\int_{-\infty}^\infty x \frac{dxdt}{x^2} = \text{ UND }\\
+\end{aligned}$$
+
+
+
+\subsection*{Question:}
+Suppose we have arrival times on $[0,\infty).$ Are they (uniform) Poisson?
+
+Let $0< T_1 <T_2<..$ be the points, compare the distribution of $(T_{k+1}-T_k)^{N}_{k=1}$ inter-arrival intervals to the exponential distribution. e.g by Kolmogorov-Smirnov test (say that 10 times fast).
+
+We discussed two approaches to this problem in class including: looking the the length between the points or sectioning it into areas to examine the distribution.
+
+\subsection*{Problem: London bombings} Let ${x_1,...,x_m}$ be points in the (2D) torus, T. Are they from a $PPP(m\times\lambda)$ i.e. constant intensity M?\\
+\textbf{Approach:} look for over-or under- dispersal.
+\begin{enumerate}
+    \item Estimate $m$: if $M \sim Pois(m) \Rightarrow \text{then } M \approx m$
+    
+    \item Estimate local density: 
+    
+    Let $\rho(r):\mathbb{R}_{\geq 0}\rightarrow \mathbb{R}_{\geq 0}$, where $\rho(r)=0$ if $n>\frac{1}{2}$, $\rho(r) \geq 0$, and $\int_{T} \rho(|x|)dx=1$.
+    
+    When $N=\sum_i \delta {x_i}$ and  $N\sim PPP(m)$, let $P_i = \sum\limits_{j\neq i} \rho (|x_j-x_i|)$ and $P=\sum_i P_i =\int_{T\times T\setminus diag(T)} e^{(|x-y|)}N(dx)N(dy)$. Then:
+
+    $$\begin{aligned}
+    \mathbb{E}[N] &= \mathbb{E}\int_{T\times T \setminus diag(t)}  \rho (|x-y|)N(dx)N(dy) \\
+    &= \int_{T\times T \setminus diag(T)} \rho  (|x-y|) \mathbb{E}[N(dx)N(dy)] \\
+    &= \lim_{\varepsilon \to 0} \frac{1}{(\pi \varepsilon^2)}^2 N(\beta_\varepsilon(x))N(\beta_\varepsilon(y)) 
```

Comment:
In these equations you switched from the equation for E[N] to the equation for E[N(dx) N(dy)]; this needs to be split out.

Reply:
Oh, also, those should be $B_\epsilon(x)$, not $\beta_\epsilon(x)$ (and should say that's the ball of radius epsilon about x).